### PR TITLE
As the issue https://github.com/entria/graphql-dataloader-boilerplate…

### LINKED ISSUE #51: Use docker to runing boilerplate
This project is very nice, but need from configuration to use docker.

I get started to change this project to use docker.

### PR DESCRIPTION
**Pull request by vcapretz**
…/issues/51 were opened requesting the docker support for this boilerplate, this commit introduces it to the project

It was added both dockerfile and docker-compose.yml so you can also run docker-compose commands to build and start the project

The config.js file was updated to support an env variable called MONGO_URL so we could pass the correct one and do not conflict with the localhost

closes #51

### CODE REVIEW

**Comment by vcapretz** [comment]
done @sibelius! :+1:

**Comment by sibelius** [comment]
tks, great work

**Comment by jgcmarins** [comment]
Good job, @vcapretz

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 *.sublime-workspace
 .idea/
 .vscode/
+package-lock.json
```
Comment (vanthome) [inline .gitignore]: According to the recommendation from NPM, this file should be checked in.

Comment (JCMais) [inline .gitignore]: we are already using `yarn.lock` for that

**Pull request merged**